CHANGELOG.md

```diff
@@ -143,6 +143,17 @@
   - Changed `.m-pagination--small` to `.m-pagination--s`
   - Changed `.m-pagination--large` to `.m-pagination--l`
 
+#### Progress
+
+##### Added
+- Added small and extra small variant
```

Comment:
Misschien beter S - M - L gebruiken (i.p.v. XS - S - M)?

Reply:
In de huidige core branding is er slecht 1 progress bar en die is 24px hoog. Daarom had ik nu die als default genomen en heb ik de twee kleinere (nieuwe) varianten als X en XS gedefinieerd. 🙂 

S - M - L is voor mij ook goed. Ik pas het aan. 🙂